crates/mithril-oxide-sys/src/IR/Operation.rs

```diff
@@ -45,6 +52,11 @@ impl ffi::Operation {
     pub fn print(self: Pin<&mut Self>) -> String {
         Operation_print(self)
     }
+
+    #[must_use]
+    pub fn get_result(self: Pin<&mut Self>, idx: u32) -> UniquePtr<OpResult> {
```

Comment:
Since we're binding the `getResult()` method, I think we should keep the C++ API's naming style.
```suggestion
    pub fn getResult(self: Pin<&mut Self>, idx: u32) -> UniquePtr<OpResult> {
```